new-content/learning-resources/tutorials.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: Tutorials
-to: https://github.com/Qiskit/qiskit-tutorials
-image: /images/landing-page/feature-hardware.jpg
-ctaLabel: Go to Khan Academy
+title: Qiskit Tutorial
+to: https://qiskit.org/documentation/tutorials/circuits/1_getting_started_with_qiskit.html
```

Comment:
On the footer, tutorials point to `https://github.com/Qiskit/qiskit-tutorials` not to the main page of circuits

Reply:
That is fine, this is pointing to the first page of the tutorials which is intended for users who select one day and advance.